### PR TITLE
Split out publications and added newsletter redirect

### PR DESCRIPTION
**Pull request by nlin21**
Split out publications from resources/documents & tutorials/ into a new directory
Add newsletter redirect to blog and front page

### CODE REVIEW

File: blog/index.html [hunk 1]
```diff
@@ -4,7 +4,11 @@
 ---
 {% for post in paginator.posts %}
     <div class="post-preview">
-        <h3><a href="{{ post.url }}">&raquo; {{ post.title }}</a></h3>
+        {% if post.title contains "Newsletter" %}
```
Comment (yuna9) [inline blog/index.html:7]: I recommend you instead check for the presence of a `link` variable in the post front matter. It doesn't have to be just newsletters

File: calendar/index.html [hunk 2]
```diff
@@ -3,4 +3,4 @@
 title: Event and Meeting Calendar
 description: You can check StuyPulse's calendar to see the meeting and event schedules.
 ---
-<iframe src="https://calendar.google.com/calendar/b/2/embed?height=600&amp;wkst=1&amp;bgcolor=%23ffffff&amp;ctz=America%2FNew_York&amp;src=aW5mb0BzdHV5cHVsc2UuY29t&amp;src=c3R1eXB1bHNlLmNvbV9vcWdxZXYyMTU5MWh0cXNqZGhoZzM0bWgza0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t&amp;color=%23D50000&amp;color=%23D81B60" style="border-width:0 overflow: hidden" width="100%" height="500" frameborder="0" scrolling="yes"> </iframe>
+<iframe src="https://calendar.google.com/calendar/b/2/embed?height=600&amp;wkst=1&amp;bgcolor=%23ffffff&amp;ctz=America%2FNew_York&amp;src=aW5mb0BzdHV5cHVsc2UuY29t&amp;src=c3R1eXB1bHNlLmNvbV9vcWdxZXYyMTU5MWh0cXNqZGhoZzM0bWgza0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t&amp;color=%23D50000&amp;color=%23D81B60" style="border-width:0; overflow: hidden" width="100%" height="500" frameborder="0" scrolling="yes"> </iframe>
```
Comment (yuna9) [inline calendar/index.html:6]: Hm why did this change?

**Review by yuna9 — CHANGES_REQUESTED** [review]
I left some comments. Also when making changes like these, it's often helpful to include screenshots of the changes. For example, you can show what the new publications page looks like and you can also show the old resources page with the publications gone.

File: calendar/index.html (hunk 2 above)
Comment (nlin21) [reply calendar/index.html:6]: image

Without the semi-colon it was returning the error in the picture, but I just tried it without the semi-colon and it still works
I'll just remove the semi-colon to revert the change

**Review by yuna9 — APPROVED** [review]

**Pull request merged**